test/integration/task-lists/ntco.js

```diff
@@ -30,7 +30,8 @@ describe('NTCO', () => {
     it('sees tasks with status "with-ntco" or "awaiting-endorsement" for their own establishment', () => {
       const expected = [
         'pil with ntco',
-        'another with-ntco to test ordering'
+        'another with-ntco to test ordering',
+        'trainingPil with ntco'
```

Comment:
It might be useful to have something to cover `isTrainingPilAtOtherEst` here as well.